package.json

```diff
@@ -119,6 +119,7 @@
     "msw": "^1.3.2",
     "msw-storybook-addon": "^1.10.0",
     "parcel": "2.11.0",
+    "posthtml-expressions": "^1.11.3",
```

Comment:
throws warning 
https://github.com/oasisprotocol/explorer/actions/runs/7886168455/job/21518858301?pr=1244#step:6:11

Reply:
Yes, it's a known issue that using JS code for posthtml kills caching of html builds.
However, for deployment, we are hosting the statically build files anyway, and for devs, the loss of speed is insignificant. (The code still builds very fast.)

So I don't think this is an issue. (Just ugly.)